app/hardware/server/list/list.index.pug

```diff
@@ -32,6 +32,7 @@ block bottom-left
             href="https://billing.synergycp.com"
             translate='server.form.no-space.UPGRADE'
             target='_blank'
+            ng-if="!app.theme.whitelabeling"
```

Comment:
we should keep this even with whitelabeling. It's OK for admins to occasionally see references to SynergyCP, it's clients that are most important to not see it.